tests/scripts/pulpcore/test_orphans.sh

```diff
@@ -6,3 +6,40 @@
 expect_succ pulp orphan cleanup
 
 test "$(echo "${OUTPUT}" | jq -r '.state' )" = "completed"
+
+pulp debug has-plugin --name "file" || exit 3
+
+if pulp debug has-plugin --name "core" --max-version "3.13.0"
+then
+  expect_fail pulp orphan cleanup --content-hrefs '["/pulp/api/v3/content/random/"]'
+else
+  dd if=/dev/urandom of=test_1.txt bs=2MiB count=1
+  dd if=/dev/urandom of=test_2.txt bs=2MiB count=1
+  dd if=/dev/urandom of=test_3.txt bs=2MiB count=1
+
+  expect_succ pulp file content upload --file test_1.txt --relative-path orphan_test/test_1.txt
+  expect_succ pulp file content upload --file test_2.txt --relative-path orphan_test/test_2.txt
+  expect_succ pulp file content upload --file test_3.txt --relative-path orphan_test/test_3.txt
+
+  content_href="$(echo "$OUTPUT" | jq -r .pulp_href)"
+  expect_succ pulp file content list
+  count="$(echo "$OUTPUT" | jq -r length)"
+  expect_succ pulp orphan cleanup --content-hrefs "[\"$content_href\"]"
+  expect_succ pulp file content list
+  if pulp debug has-plugin --name "core" --min-version "3.15.0"
+  then
+    test "$(echo "$OUTPUT" | jq -r length)" -eq "$count"
```

Comment:
Wouldn't this branch be a bug in pulpcore 3.14?

Reply:
Yeah you're right, I had to find the file under .ci/settings/settings.py, but we do set protection time to 0 for the tests. Changed the tests to accommodate for that.